src/Icons/src/Twig/UXIconRuntime.php

```diff
@@ -45,6 +46,10 @@ public function renderIcon(string $name, array $attributes = []): string
             }
 
             throw $e;
+        } catch (TransportException $e) {
+            $this->logger?->warning($e->getMessage());
+
+            return '';
         }
```

Comment:
We could merge this block with the `catch (IconNotFoundException $e)`

Reply:
> the option set to false, so we make it explicit
the option set to true in when@dev

IMO, `ignoreNotFound` should be `true` to prevent production to return a 500 when an icon is missing.
And `false` in `when@dev`

This is why that option was originally added: https://github.com/symfony/ux/issues/2008

Reply:
Ah, yes indeed, I forgot what was the option name, but yes these errors should be silent in prod